spatialos-sdk/src/worker/commands.rs

```diff
@@ -6,31 +6,31 @@ use spatialos_sdk_sys::worker::{
 };
 
 // TODO: Wrap Schema_CommandRequest
-pub struct CommandRequest {
+pub struct CommandRequest<'a> {
     pub component_id: ComponentId,
-    pub schema_type: *mut Schema_CommandRequest,
+    pub schema_type: &'a Schema_CommandRequest,
```

Comment:
Note that this is going to conflict pretty hard with my branch :P

Reply:
I figured as much. I'm down to address conflicts once your changes get merged in :+1: